README.md

```diff
@@ -283,6 +283,16 @@ const agenda = new Agenda({maxConcurrency: 20});
 Takes a `number` which specifies the default number of a specific job that can be running at
 any given moment. By default it is `5`.
 
+The concurrency parameter is only important when the job takes longer than its
```

Comment:
Pretty sure this is incorrect. Gonna throw together a minimal repo to try and confirm.

Reply:
Ok, I've thrown this together: https://github.com/wingsbob/agenda-concurrency-example

You can run with different default concurrency levels by setting the environment variable `CONCURRENCY`. The lock time is left as the default (10 mins), so that's definitely not coming into effect.

`Agenda#every` might not be the best example for concurrency, as it creates `single` type jobs, which seem to be semi-unique